### DISCUSSION TITLE
Kata 2

### DISCUSSION THREAD

**Discussion by TorresTorresMJ** [LaunchX-InnovaccionVirtual/CursoIntroPython#59] [Q&A] [answered]
Hola, la verdad no entiendo porque no me permite  desde la terminal ir al directorio, 
![image](https://user-images.githubusercontent.com/99225303/153319998-077be042-39d2-422b-930b-7f98f6b584b8.png)

inclusive abrí desde la carpeta la dirección en Windows terminal, y la puse tal cual y seguí la instrucción en el error, pero no me lo importa. Si alguien llega a ver mi error que yo no he logrado ver, se lo agradezco si me dice, ¡gracias!

**Comment by OscarPedroza** [comment, marked as answer]
Hola, para que desde la terminal te deje acceder al directorio utiliza el comando `cd` y después la ruta, sería algo como: `cd Documents\PythonLaunchX`

**Reply by TorresTorresMJ** [reply]
Awesome! Muchísimas gracias Oscar
